templates/takeovers/_active-directory_takeover.html

```diff
@@ -0,0 +1,44 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row u-clearfix u-vertically-center">
+    <div class="col-7 ">
+      <h1>
+        Integrate Ubuntu Desktop with Active Directory
+      </h1>
+      <h4 class="p-heading--four">
+        A step by step guide to joining your desktops to Active Directory
+        domains.
+      </h4>
+      <a href="/engage/microsoft-active-directory?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_IOT_Desktop_Whitepaper_ActiveDirectory" class="p-button--positive u-hide--small"
+        ><span>Download the whitepaper</span></a
+      >
```

Comment:
the `<a>` tag is broken up unnecessarily here (compare to the `<a>` tag on line 24)